tests/MongoDB.Driver.Tests/ClusterRegistryTests.cs

```diff
@@ -22,9 +22,11 @@
 using System.Security.Cryptography.X509Certificates;
 using FluentAssertions;
 using MongoDB.Bson;
+using MongoDB.Bson.TestHelpers;
```

Comment:
There are a number of namespaces(added in previous tickets) which are not used in this class. They can be removed if you don't mind

Reply:
Done.